.eslintignore

```diff
@@ -1 +1,2 @@
-example
\ No newline at end of file
+example
+**/node_modules/*
```

Comment:
Not sure why this was needed for health-check package, but other packaged managed to run with the default eslint configuration without desdencing linting node_modules.

Reply:
no worries, this is reasonable